docs/source/troubleshooting.rst

```diff
@@ -226,9 +226,32 @@ On Ubuntu / Debian:
 .. code-block:: bash
 
     /usr/lib/python2.7/dist-packages/st2common/bin/st2-self-check
-    
+
 On RedHat / Fedora:
 
 .. code-block:: bash
 
     /usr/lib/python2.7/site-packages/st2common/bin/st2-self-check
+
+
+Support
+-------
+
+Support is available to you via `email <support@stackstorm.com/>`_ or via
+`slack-community <https://stackstorm.typeform.com/to/K76GRP)/>`_.
+
```

Comment:
What is the right link to give for https://stackstorm-community.slack.com? 

if we give https://stackstorm.typeform.com/to/K76GRP this is a sign-up doestn't work for who alraedy signed up. 
if we give  https://stackstorm-community.slack.com - no link back to sign up...